src/sql/parts/grid/views/query/query.component.ts

```diff
@@ -92,7 +92,8 @@ export class QueryComponent extends GridParentComponent implements OnInit, OnDes
 			hoverText: () => { return LocalizedConstants.saveCSVLabel; },
 			functionality: (batchId, resultId, index) => {
 				let selection = this.getSelection(index);
-				if (selection.length <= 1) {
+				let columns = selection[0].toCell - selection[0].fromCell + 1;
```

Comment:
Selection is the group of rows from a grid. So, it's safe to take `selection[0]` for columns.